test/mpi/maint/coll_cvars.txt

```diff
@@ -236,6 +236,7 @@ algorithms:
             composition:4
             smp
             recursive_doubling
+            recursive_multiplying
```

Comment:
Add parameters to test non-default `k`. See the `tree` algorithm below.

Reply:
Fixed